Makefile

```diff
@@ -5,3 +5,11 @@ PHONY: \
 
 build_test_image:
 	$(DOCKER_COMPOSE) -f docker-compose.yml  run --rm postgres -d
+
+test:
+	go generate ./...
+	docker-compose -f testdata/docker-compose.yml up --abort-on-container-exit --remove-orphans
+
+test_fileoutput:
+	go generate ./...
+	docker-compose -f testdata/docker-compose.yml up --abort-on-container-exit --remove-orphans > testing.txt
```

Comment:
Do we really need it?
I mean… we're hardcoding a file name `testing.txt` that user might prefer to be somewhere else. Also, this file is not on `.gitignore`

Have you considered removing `test_fileoutput` and instead using `make test > MY_FILE_PATH`?

I see this way as a better option:
- we keep `.git` save by not producing files that should not be committed
- we avoid polluting `.gitignore`
- we give the user freedom to create this file else where (e.g. in `/tmp`, that would be my choice)

The only trade-off I see is that it `make test > /tmp/test-output` would include the `go generate ./...` — but overall I feel like a good trade off. What do you think?

Reply:
It looks good to me!